{{cookiecutter.project_name}}/notebooks/TriggerModelDeploy.py

```diff
@@ -45,15 +45,24 @@
 assert env != "None", "env notebook parameter must be specified"
 assert model_uri != "", "model_uri notebook parameter must be specified"
 
-{% if cookiecutter.cicd_platform == "gitHub" -%}
+{% if cookiecutter.cicd_platform in ["gitHub", "gitHubEnterprise"] -%}
+github_server = dbutils.secrets.get(
+    f"{env}-{{cookiecutter.project_name}}-cd-credentials", "github_server"
+)
 github_repo = dbutils.secrets.get(
     f"{env}-{{cookiecutter.project_name}}-cd-credentials", "github_repo"
 )
 token = dbutils.secrets.get(
     f"{env}-{{cookiecutter.project_name}}-cd-credentials", "token"
 )
-cd_trigger_url = f"https://api.github.com/repos/{github_repo}/actions/workflows/deploy-model-{env}.yml/dispatches"
+if github_server == "https://github.com" or github_server == "github.com":
```

Comment:
Might be safer to use `cookiecutter.cicd_platform` here as well instead of bifurcating on URLs (someone could use a `http` or throw in a `www`)